<ui version="4.0" >
 <class>TabRev</class>
 <widget class="QWidget" name="TabRev" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Rev list</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="horizontalSplitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="ListView" name="listViewLog" >
      <property name="contextMenuPolicy" >
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights" >
       <bool>true</bool>
      </property>
      <property name="itemsExpandable" >
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSplitter" name="verticalSplitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabLogDiff" >
          <property name="tabPosition" >
           <enum>QTabWidget::East</enum>
          </property>
          <property name="tabShape" >
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="Log" >
           <attribute name="title" >
            <string>Log</string>
           </attribute>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="RevDesc" name="textBrowserDesc" >
              <property name="lineWrapMode" >
               <enum>QTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Diff" >
           <attribute name="title" >
            <string>Diff</string>
           </attribute>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="PatchContent" name="textEditDiff" >
              <property name="documentTitle" >
               <string comment="testing with diff view" >test</string>
              </property>
              <property name="undoRedoEnabled" >
               <bool>false</bool>
              </property>
              <property name="lineWrapMode" >
               <enum>QTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="frameLogDiff" >
          <widget class="QWidget" name="page" />
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="FileList" name="fileList" >
       <property name="contextMenuPolicy" >
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>ListView</class>
   <extends>QTreeView</extends>
   <header>listview.h</header>
  </customwidget>
  <customwidget>
   <class>RevDesc</class>
   <extends>QTextBrowser</extends>
   <header>revdesc.h</header>
  </customwidget>
  <customwidget>
   <class>FileList</class>
   <extends>QListWidget</extends>
   <header>filelist.h</header>
  </customwidget>
  <customwidget>
   <class>PatchContent</class>
   <extends>QTextEdit</extends>
   <header>patchcontent.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >revdesc.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
